### ISSUE TITLE
String with slashes no longer parsed correctly in 3.4.1

### ISSUE THREAD

**Issue by Fjan** [rails/sprockets-rails#487]
In sprockets-rails version 3.4.0 this CSS works correctly:
```scss
$cdn: "//cdn.example.net";
background: url(#{$cdn}/icon/icn.png) no-repeat;
```

In sprockets-rails version 3.4.1 one of the slashes in the URL disappears from the output. As a mitigation you can quote the string inside the url function, then the slashes appear unharmed.

**Comment by rafaelfranca** [comment]
@jcoyne this one is also related to the absoute path fix.

**Comment by rafaelfranca** [comment]
@Fjan Can you check if this issues was fixed by #485?

**Comment by Fjan** [comment]
Confirmed, that fixes it. Thanks!
